src/language-extractor.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * Created by Anshul on 3/7/2017.
+ */
```

Comment:
Yeah, these were added by Intellij automatically when creating the file. I'll remove these.